examples/quick-start-artillery/tracetest-provision.yaml

```diff
@@ -0,0 +1,27 @@
+---
```

Comment:
Does the user need to run a command with the CLI to configure the provision or will it happen on-startup?

eg.

```
tracetest apply -f ./tracetest-provision.yaml
```

If it is needed, we might need to specify this?